.travis.yml

```diff
@@ -9,7 +9,6 @@ php:
   - 7.2
   - 7.3
   - 7.4
-  - nightly
```

Comment:
Why removing nightly (php8)?

Reply:
I would instead remove 7.3 and 7.4 to save some workers until both 7.2 and 8.0 work.

Reply:
It's temporary) I commented out almost all combinations to faster tests passing. Since agnostic tests became green I will return all things back.